drivers/bluetooth/hci/Kconfig.infineon

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Change size of command lengths. It for vendor commands related to
+# firmware downloading.
+config BT_BUF_CMD_TX_SIZE
+	default 255
+
+choice  CYW43XXX_PART
+	prompt "Select CYW43XXX part"
+	depends on BT_CYW43XXX
+
+config CYW4343W
+	bool "CYW4343W"
+	help
+	  Enable Infineon CYW4343W BLE connectivity,
+	  More information about CYW4343W device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw4343w/
+
+config CYW4373
+	bool "CYW4373"
+	help
+	  Enable Infineon CYW4373 BLE connectivity,
+	  More information about CYW4373 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw4373/
+
+config CYW43012
+	bool "CYW43012"
+	help
+	  Enable Infineon CYW43012 BLE connectivity,
+	  More information about CYW43012 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw43012/
+
+config CYW43438
+	bool "CYW43438"
+	help
+	  Enable Infineon CYW43438 BLE connectivity,
+	  More information about CYW43438 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw43438/
+
+config CYW43439
+	bool "CYW43439"
+	help
+	  Enable Infineon CYW43439 BLE connectivity,
+	  More information about CYW43439 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw43439/
+
+
+choice CYW43012_MODULE
+	prompt "Select CYW43012 module"
+	depends on CYW43012
+	depends on BT_CYW43XXX
+
+config CYW43012_MURATA_1LV
+	bool "MURATA-1LV"
+endchoice
+
+choice CYW4343W_MODULE
+	prompt "Select CYW4343W module"
+	depends on CYW4343W
+	depends on BT_CYW43XXX
+
+config CYW4343W_MURATA_1DX
+	bool "MURATA-1DX"
+endchoice
+
+choice CYW4373_MODULE
+	prompt "Select CYW4373 module"
+	depends on CYW4373
+	depends on BT_CYW43XXX
+
+config CYW4373_STERLING_LWB5PLUS
+	bool "STERLING-LWB5plus"
+endchoice
+
+choice CYW43439_MODULE
+	prompt "Select CYW43439 module"
+	depends on CYW43439
+	depends on BT_CYW43XXX
+
+config CYW43439_MURATA_1YN
+	bool "MURATA_1YN"
+endchoice
+
+config CYW43XX_CUSTOM_FIRMWARE_HCD_BLOB
+	string  "Path to user BT firmware HCD file"
+	help
+	  Path to BT firmware HCD file for custom or vendor CYW43xx modules.
+	  It can be absolute path, or relative from project folder.
+
+endchoice
```

Comment:
I think this should be closed earlier? Looks like you are having two `choice` nested into each other and it looks weird in menuconfig.

Reply:
updated